tests/e2e/accountListFiltering.spec.ts

```diff
@@ -42,35 +42,41 @@ describe("Account creation, edition and deletion", function () {
         cy.get("div").contains(`@${adminAuth.username}`);
     });
 
-    it("Role filter buttons exist", function () {
-        cy.get("button[id='role-All']").should("exist");
+    it("Role filter select exist", function () {
+        cy.get("button[id='selectFilterRole']").should("exist");
+        cy.get("button[id='selectFilterRole']").click();
```

Comment:
The `should("exist")` is implicitly included in the `click`
```suggestion
        cy.get("button[id='selectFilterRole']").click();
```